invenio_records_resources/services/records/schema.py

```diff
@@ -53,12 +41,42 @@ def clean(self, data, **kwargs):
         return data
 
 
+class TypeSchema(Schema):
```

Comment:
Question: What's the reason for adding the type schema? As this is data model specific I would not add it here.

I aware the others below should like ``metadata.title`` should likely not be here either.

Reply:
The faceting tests in records-resources index a `type` field with children `type` + `subtype`. By removing the `INCLUDE` above, those needed to be added explicitly. I do agree that in fact, just `BaseRecordSchema` should be defined in `schema.py` and that tests (`mock module`) should be the one defining the `RecordSchema` with title + type. If that makes sense, I can do that change?

Reply:
Did the change :nerd_face: